<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperazioniDlg</class>
 <widget class="QDialog" name="OperazioniDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>291</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>304</width>
    <height>291</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestione Cassa</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>MODALITA'</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="testBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>71</height>
         </size>
        </property>
        <property name="text">
         <string>Modalit√† Test</string>
        </property>
        <property name="icon">
         <iconset resource="risorse.qrc">
          <normaloff>:/GestCassa/warning</normaloff>:/GestCassa/warning</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="cassaBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>71</height>
         </size>
        </property>
        <property name="text">
         <string>Cassa</string>
        </property>
        <property name="icon">
         <iconset resource="risorse.qrc">
          <normaloff>:/GestCassa/cassa</normaloff>:/GestCassa/cassa</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="gestioneBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>71</height>
         </size>
        </property>
        <property name="text">
         <string>Gestione</string>
        </property>
        <property name="icon">
         <iconset resource="risorse.qrc">
          <normaloff>:/GestCassa/gestione</normaloff>:/GestCassa/gestione</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>FUNZIONI AMMINISTRATIVE</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="reportBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>71</height>
         </size>
        </property>
        <property name="text">
         <string>Report</string>
        </property>
        <property name="icon">
         <iconset resource="risorse.qrc">
          <normaloff>:/GestCassa/report 2</normaloff>:/GestCassa/report 2</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="configurazioneBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>71</height>
         </size>
        </property>
        <property name="text">
         <string>Configurazione</string>
        </property>
        <property name="icon">
         <iconset resource="risorse.qrc">
          <normaloff>:/GestCassa/configurazione</normaloff>:/GestCassa/configurazione</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>FUNZIONI COMUNI</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="statsBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>71</height>
         </size>
        </property>
        <property name="text">
         <string>Statistiche</string>
        </property>
        <property name="icon">
         <iconset resource="risorse.qrc">
          <normaloff>:/GestCassa/stats</normaloff>:/GestCassa/stats</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="stornoBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>71</height>
         </size>
        </property>
        <property name="text">
         <string>Storno</string>
        </property>
        <property name="icon">
         <iconset resource="risorse.qrc">
          <normaloff>:/GestCassa/cancella</normaloff>:/GestCassa/cancella</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="closeBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>71</height>
         </size>
        </property>
        <property name="text">
         <string>Esce</string>
        </property>
        <property name="icon">
         <iconset resource="risorse.qrc">
          <normaloff>:/GestCassa/esce</normaloff>:/GestCassa/esce</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="risorse.qrc"/>
 </resources>
 <connections/>
</ui>
